interview-questions.md

```diff
@@ -8,63 +8,112 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is a function? Why would you use one?
 
   Your answer:
+    the anatomy of a function -
+      const myFunction = (argument) => {
+        logic
+      }
+    to evoke a function in console.log - 
+      console.log(myFunction(parameter))
```

Comment:
This is very cool. But I would encourage you to really think about how you would explain this in person if you were over phone or face to face without text available for you to show syntax. 

Additionally, I love the want to explain EVERY aspect of the function, but in an interview setting, this would be VERY verbose.  Your answer is GOOD, but I feel this would be just slightly long winded is all.